packages/expo-av/android/src/main/java/expo/modules/av/player/SimpleExoPlayerData.java

```diff
@@ -123,6 +123,7 @@ public void load(final Bundle status, final LoadCompletionListener loadCompletio
   public synchronized void release() {
     if (mSimpleExoPlayer != null) {
       mSimpleExoPlayer.release();
+      stopUpdatingProgressIfNecessary();
       mSimpleExoPlayer = null;
     }
```

Comment:
```suggestion
    stopUpdatingProgressIfNecessary();
    if (mSimpleExoPlayer != null) {
      mSimpleExoPlayer.release();
      mSimpleExoPlayer = null;
    }
```

just a nit to align with https://github.com/expo/expo/blob/b241fb39b6dfd4fc804792651d126594ba9fdd30/packages/expo-av/android/src/main/java/expo/modules/av/player/MediaPlayerData.java#L144-L152
i am not quite familiar with expo-av yet. please do whatever you think is better.